packages/projection/src/operators/Mappers/util.ts

```diff
@@ -1,11 +1,14 @@
 import { Asset, Cardano, Handle } from '@cardano-sdk/core';
-import { InvalidStringError } from '@cardano-sdk/util';
+import { Logger } from 'ts-log';
 
 /** Up to 100k transactions per block. Fits in 64-bit signed integer. */
 export const computeCompactTxId = (blockHeight: number, txIndex: number) => blockHeight * 100_000 + txIndex;
 
-export const assetNameToUTF8Handle = (assetName: Cardano.AssetName): Handle => {
+export const assetNameToUTF8Handle = (assetName: Cardano.AssetName, logger: Logger): Handle | null => {
   const handle = Cardano.AssetName.toUTF8(assetName);
-  if (!Asset.util.isValidHandle(handle)) throw new InvalidStringError(`Invalid handle ${handle}`);
+  if (!Asset.util.isValidHandle(handle)) {
+    logger.warn(`Invalid handle: '${handle}' / '${assetName}'`);
+    return null;
+  }
```

Comment:
:+1: 

Just wondering if we wish to do something with these invalid handles. I kinda wish that we had some queue in this project, we could just submit it to it and at some point review whatever invalid handles we find

Reply:
Yes, that would be nice, but it's a bigger task. This will at least unblock deployments for now.